### PR TITLE
Merge code from SPU-project-form-handler to SPU-view-projects

### PR DESCRIPTION
**Pull request by TrNgTien**
1. So, I merge the mock-up to SPU-view-projects and and i will switch the branch and i will make the action push data to Firebase
2. Branch SPU-projects-form-handler will use for sending the data application form of SPU to Gmail of PPUs
3. I will duo with Long in this branch, detail: Me and Long will do in 1 file action

### CODE REVIEW

File: src/pages/app/searching_project_user/ProjectInfoPage.js [hunk 1]
```diff
@@ -219,6 +219,7 @@ const ProjectInfoPage = (props) => {
           ))}
         </ul>
       </div>
+      //Somthing is wrong in this section, detail:"benefits". That makes the page broken
```
Comment (TrNgTien) [inline src/pages/app/searching_project_user/ProjectInfoPage.js:222]: @kenpham4real 
plz resolve that asap, so i can code :"(((, becuz my task is inherited by this page :"((

File: src/pages/app/posting_project_user/styles/BeautifyProfileStyles.css [hunk 2]
```diff
@@ -39,7 +39,6 @@
     border:2px solid black;
     border-radius: 7px;
 }
-<<<<<<< HEAD
 .avatar-box{
     border:2px solid black;
     border-radius: 5px;
```
Comment (kenpham4real) [inline src/pages/app/posting_project_user/styles/BeautifyProfileStyles.css]: The whole style is already separated for the `<ImagePreview/>` component
Please remove this style
If you want to use the image preview feature, just import it from the **component** folder

**Review by kenpham4real — CHANGES_REQUESTED** [review]
The image preview has already been separated to be a component. Check it in the folder
The component has it's style already. Please edit in that file and remove these redundant code

File: src/pages/app/searching_project_user/ProjectInfoPage.js (hunk 1 above)
Comment (kenpham4real) [reply src/pages/app/searching_project_user/ProjectInfoPage.js:222]: I'm working on it now.

File: src/pages/app/posting_project_user/styles/BeautifyProfileStyles.css (hunk 2 above)
Comment (TrNgTien) [reply src/pages/app/posting_project_user/styles/BeautifyProfileStyles.css]: ok got it

File: src/pages/app/searching_project_user/ApplyForm.js [hunk 3]
```diff
@@ -5,12 +5,62 @@
...
+    }
+
+
+    /**
+     * @summary Handle State of Message
+     * @param {string} Message
+     * @return {void}
+     */
+    const onChangeMessage=(Message) =>{
+        setMessage(Message);
```
Comment (kenpham4real) [inline src/pages/app/searching_project_user/ApplyForm.js:57]: @TrNgTien you seem to misunderstand the MVP flow.
The MVP contains many inputs for each question which are fetched from Long's actions

**Review by kenpham4real — CHANGES_REQUESTED** [review]

**Pull request merged**